packages/frontend/stalker-app/src/app/modules/domains/list-domains/list-domains.component.ts

```diff
@@ -316,4 +334,58 @@ export class ListDomainsComponent {
     event.stopPropagation();
     this.startDate = new Date(Date.now() - defaultNewTimeMs);
   }
+
+  block() {
+    const bulletPoints = this.getSelectionAsBulletPoints();
+
+    let data: ConfirmDialogData;
+    if (bulletPoints.length > 0) {
+      const block = async (block: boolean) => {
+        try {
+          await this.domainsService.block(
+            this.selection.selected.map((s) => s._id),
+            block
+          );
+          this.selection.clear();
+          this.toastr.success(
+            block
+              ? $localize`:Domains blocked|Blocked a domain:Domains blocked successfully`
+              : $localize`:Domains unblocked|Unblocked a domain:Domains unblocked successfully`
```

Comment:
Je crois que le format suivant est plus commun

```suggestion
              ? $localize`:Domains blocked|Blocked a domain:Domains successfully blocked`
              : $localize`:Domains unblocked|Unblocked a domain:Domains successfully unblocked`
```